docs/shared/core-features/enforce-project-boundaries.md

```diff
@@ -131,7 +131,31 @@ Next you should update your root lint configuration:
 
 With these constraints in place, `scope:client` projects can only depend on other `scope:client` projects or on `scope:shared` projects. And `scope:admin` projects can only depend on other `scope:admin` projects or on `scope:shared` projects. So `scope:client` and `scope:admin` cannot depend on each other.
 
-Projects without any tags cannot depend on any other projects. If you add the following, projects without any tags will be able to depend on any other project.
+Projects without any tags cannot depend on any other projects, unless you allow all tags (see below).
```

Comment:
This should not be changing.

Project with no tag should not be allowed to depend on any other project.